src/components/booth.tsx

```diff
@@ -18,5 +18,8 @@ export default function Booth({location, zoom, center}: {location: TRCLocation,
         width: location.width/ftPerPixel,
         height: location.width/ftPerPixel,
         boxSizing: "border-box",
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center'
```

Comment:
Add a trailing comma here. Having a trailing comma on the last line prevents future changes from having merge conflicts.

Reply:
Use double quotes instead of single quotes to match the existing style in the file.

Reply:
> Use double quotes instead of single quotes to match the existing style in the file.

or alternatively switch `boxSizing: "border-box"` to use single quotes. I often mix them up but now that we're collaborating we gotta be consistent.